### DISCUSSION TITLE
Add Wiremock Module

### DISCUSSION THREAD

**Discussion by julien92** [testcontainers/testcontainers-java#6443] [Ideas]
I would like to contribute by adding a module for Wiremock. 

I have noticed that you already have a module for Mockserver, i think adding wiremock could increase testcontainer popularity.

Indeed wiremock is often used by the java community and they don't have a wiremock compatible with the latest Spring boot version without adding older import.

For uprading my application to the latest Spring boot version, I decided to transform wiremock into a container.

If you are interested i can do a pull request to add a new module.

regards

**Comment by eddumelendez** [comment]
Hi @julien92 what's your proposal about Wiremock module? Don't you think using `GenericContainer` is easy enough?

**Reply by julien92** [reply]
Yes GenericContainer but i see you do a module too for MockServer for whom GenericContainer is also enough.

That's why I wondered if you were interested in having the same with WireMock to provide an alternative to MockServer.

**Reply by eddumelendez** [reply]
Is there something about tls in mockserver that could require additional config and become useful to have a module?

**Reply by eddumelendez** [reply]
@julien92 Please, check https://github.com/wiremock/wiremock-testcontainers-java
